tests/MongoDB.Driver.Core.Tests/Core/Operations/EndTransactionOperationTests.cs

```diff
@@ -96,14 +99,19 @@ public void CommandName_should_return_expected_result()
         [Theory]
         [ParameterAttributeData]
         public void CreateCommand_should_return_expected_result(
+            [Values(true, false)] bool useRecoveryToken,
             [Values(1, 2)] int w)
         {
             var writeConcern = new WriteConcern(w);
-            var subject = CreateSubject(writeConcern: writeConcern);
+            var recoveryToken = BsonDocument.Parse("{ cake: false }");
+            var subject = useRecoveryToken ? CreateSubject(recoveryToken, writeConcern) : CreateSubject(writeConcern: writeConcern);
 
             var result = subject.CreateCommand();
 
-            result.Should().Be($"{{ fakeOperation : 1, writeConcern : {{ w : {w} }} }}");
+            var expected = useRecoveryToken
```

Comment:
Naming suggestion: rename expected to expectedResult.

Reply:
Fixed